src/dragonball/src/vcpu/vcpu_impl.rs

```diff
@@ -216,7 +216,7 @@ pub enum VcpuResponse {
 
 /// List of events that the vcpu_state_sender can send.
 pub enum VcpuStateEvent {
-    /// (result, response) for hotplug, result 0 means failure, 1 means success.
+    /// (result, response) for hotplug, result 0 means success, 1 means failure.
     Hotplug((i32, u32)),
```

Comment:
Thanks, added.

Reply:
I would suggest adding this
```Rust
const HotplugSuccess: i32 = 0;
const HotplugFailed: i32 = 1;
```

Reply:
Normally returning value 0 means success, is defining it as const unnecessary?

Reply:
@liubin, I think this is a nit, not defining those consts will definitely be fine. My opinions are as follows.

In C I would say this would be unnecessary, since that is the convention.

But here we have already put the `Hotplug` as a product type of i32 and u32, Things like
```Rust
if let Hotplug((x, y)) == some_func_returns_hotplug() {
        match x {
            0 => handle_success(),
            1 => handled_failure(),
        }
}
```
would be hard to read, and prone to errors(by developers) if multiple functions use this.

Moreover, if all `VcpuStateEvent`s are gonna use 0 and 1 as sentinel values, `VcpuStateEventSuccess` would be a better name.